src/sys/reboot.rs

```diff
@@ -0,0 +1,44 @@
+use {Errno, Error, Result};
+use libc;
+use void::Void;
+use std::mem::drop;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
Could you switch Clone and Copy? (I know it's pedantic, I'll provide more substantial feedback momentarily)